### PR TITLE
Fix issue when the title is an empty string

### LINKED ISSUE #7198: When the title set to string empty place for title  is still exist
### Description

When the title set to string empty is still exist
See:
image
[MauiApp25.zip](https://github.com/dotnet/maui/files/8695165/MauiApp25.zip)

### Steps to Reproduce

1.I attached a demo.
2.Click on main page and move to NewPage, the title is empty but the place is still
keep for title. On main window when title not set we not see the place for title like screenshot.

### Version with bug

Release Candidate 3 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

### Affected platform versions

NO

### Did you find any workaround?

NO, How we get rid of this title?

### Relevant log output

```shell
No


```

### PR DESCRIPTION
**Pull request by Egaros**
### Description of Change

Fix case when the title set to string empty is still exist

### Issues Fixed

Fixes  #7198

### CODE REVIEW

**Review by jsuarezruiz — APPROVED** [review]

**Review by mattleibow — APPROVED** [review]

**Review by PureWeen — CHANGES_REQUESTED** [review]

File: src/Controls/src/Core/ShellToolbar.cs [hunk 1]
```diff
@@ -113,14 +113,14 @@ internal void ApplyChanges()
 			{
 				var flyoutBehavior = (_shell as IFlyoutView).FlyoutBehavior;
 #if WINDOWS
-				IsVisible = (BackButtonVisible ||
+				IsVisible = ((BackButtonVisible && !String.IsNullOrEmpty(Title)) ||
```
[the remark this answers is not in the document]
Comment (Egaros) [reply src/Controls/src/Core/ShellToolbar.cs:116]: Hi,

According to issue #7198, the fact that the title is empty has been reported as an issue.

This happened when the back button was visible, but it is possible that I was wrong when understanding what to do to fix this bug.

[the remark this answers is not in the document]
Comment (PureWeen) [reply src/Controls/src/Core/ShellToolbar.cs:116]: You're correct @Egaros 

I wasn't thinking about this correctly in terms of windows. 
I fixed this in a slightly different way here
https://github.com/dotnet/maui/pull/7932

And added a test

**Comment by PureWeen** [comment]
Alternate PR
https://github.com/dotnet/maui/pull/7932

File: src/Controls/src/Core/ShellToolbar.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (Egaros) [reply src/Controls/src/Core/ShellToolbar.cs:116]: Oh ok, I see, thanks

**Pull request closed without merging**